src/azure-cli/azure/cli/command_modules/cognitiveservices/custom.py

```diff
@@ -9,13 +9,12 @@
 from knack.util import CLIError
 from knack.log import get_logger
 
-from azure.mgmt.cognitiveservices.models import Account, Sku,\
+from azure.mgmt.cognitiveservices.models import Account as CognitiveServicesAccount, Sku,\
     VirtualNetworkRule, IpRule, NetworkRuleSet, NetworkRuleAction,\
-    AccountProperties, ApiProperties,\
-    Identity, ResourceIdentityType
+    AccountProperties as CognitiveServicesAccountProperties, ApiProperties as CognitiveServicesAccountApiProperties,\
+    Identity, ResourceIdentityType as IdentityType
```

Comment:
@yangyuan  excuse me, why import CognitiveServicesAccount from models? Isn't this model replaced by Account in track2?